Sources/SnowManLogicSpecs/ViewSpec.swift

```diff
@@ -0,0 +1,56 @@
+import SnowManLogic
+import Swiftest
+
+class ViewSpec: Swiftest.Spec {
+  	let spec = describe("View logic") {
+  		var ui: MockUI!
+        var mockView: MockView!
+        before() {
+            ui = MockUI()
+            mockView = MockView(ui: ui)
+        }
+        
+        describe("#receiveGuess") {
+            it("should return the user input if it is a string of letters") {
+                ui.getUserInputReturn = "hello"
+                expect(mockView.receiveGuess()).to.equal("hello")
+            }
+
+            it("should return the user input if it is a single letter") {
+                ui.getUserInputReturn = "a"
+                expect(mockView.receiveGuess()).to.equal("a")
+            }
+
+            it("should return an error message if the user input is something other than a letter or word") {   
+                ui.getUserInputReturn = "1"
+                expect(mockView.receiveGuess()).to.equal("Invalid")
+            }
```

Comment:
Probably no special case for it since you're treating it as a guess of the word.